website/static/js/projectorganizer.js

```diff
@@ -775,12 +454,26 @@ function _poLoadOpenChildren() {
  * @this Treebeard.controller
  * @private
  */
+// TODO : Cleanup
 function _poMultiselect(event, tree) {
     var tb = this,
         selectedRows = filterRowsNotInParent.call(tb, tb.multiselected),
         someItemsAreFolders,
         pointerIds;
-    if (selectedRows.length > 1) {
+    _toolbarDismissEvent.call(tb);
+    tb.options.iconState.rowIcons = [];
```

Comment:
It's difficult to distinguish which of these state variables are treebeard specific and which are fangorn specific. We could add them to a namespace, e.g. `tb.options.fangorn.iconState`, but then we'd have to check if `tb.options.fangorn` in a bunch of places. We can at least add a prefix, e.g. `tb.options.fgIconState`.